lessonbuilder/tool/src/webapp/templates/BltiPicker.html

```diff
@@ -162,26 +98,27 @@ <h5 class="modal-title" id="bltiModalLabel">Modal title</h5>
       });
     }
 
-// See if we are supposed to launch the shortcut link
-var shortcut_tag = document.getElementById('app-store-shortcut::blti-create::blti-create-link');
-if ( shortcut_tag ) {
-    var shortcut_title = shortcut_tag.getAttribute('title');
-    var shortcut_href = shortcut_tag.getAttribute('href');
-    if ( shortcut_href && shortcut_title ) {
-        document.getElementById('sakai-basiclti-admin-iframe').src = shortcut_href;
-        showIframe(shortcut_title);
-    }
-}
-
-//]]>
 
-//If none of the tools have a description, hides the column.
 $( document ).ready(function() {
-    var emptyDescriptionCount = $('.externalToolContainerClass td:nth-child(2):not(:has(:not(:empty)))').length;
-    var rowNumber = $('.externalToolContainerClass tbody tr').length;
-    if(emptyDescriptionCount == rowNumber){
-        $('.externalToolContainerClass td:nth-child(2), .externalToolContainerClass th:nth-child(2)').hide();
-    }
+    console.log("READY");
+		// If we have the blti-autosubmit form, submit it therwise show picker
+		setTimeout(function(){
+			var form = document.getElementById("blti-autosubmit");
+            if ( form != null ) {
+                form.submit();
+            } else {
+                console.log("There is no form to submit");
```

Comment:
console.debug?

Reply:
No - this is a really profound screw up.  It means things are very broken.  What will happen next is failure so we might as well have a serious log message.